package/Dockerfile-xen

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV USER root
+
+# build depends
+RUN mkdir -p /log && \
+    sed -i 's/# deb-src/deb-src/g' /etc/apt/sources.list && \
+    apt-get update && \
+    echo Running apt-get install, this may take a few minutes... && \
+    apt-get --quiet --yes install wget git bcc bin86 gawk bridge-utils iproute2 libcurl4-openssl-dev bzip2 pciutils-dev build-essential make gcc clang libc6-dev libc6-dev-i386 linux-libc-dev zlib1g-dev libncurses5-dev patch libvncserver-dev libssl-dev iasl libbz2-dev e2fslibs-dev git-core uuid-dev ocaml libx11-dev bison flex ocaml-findlib xz-utils gettext libyajl-dev libpixman-1-dev libaio-dev libfdt-dev cabextract libglib2.0-dev autoconf automake libtool libjson-c-dev libfuse-dev liblzma-dev autoconf-archive kpartx python-dev cmake libsdl-dev && \
```

Comment:
For some reason, build fails when this long line gets replaced with `apt-get --quiet --yes install wget && apt-get --quiet --yes build-dep xen xen-tools`. TODO investigate further

Reply:
The package is called `xen-hypervisor-4.9-amd64` on 18.04: https://packages.ubuntu.com/search?suite=bionic&searchon=names&keywords=xen-hypervisor

Reply:
I guess you should do `build-dep xen-utils-4.9 xen-hypervisor-4.9-amd64`

Reply:
When I type `xen-utils-4.9 xen-hypervisor-4.9-amd64` it's telling something like "using `xen` as source package instead" for both of these, so it doesn't change anything at all.

Turns out that everything works if I additionally install `build-essential git wget flex bison` (+ LibVMI/DRAKVUF dependencies).